x-pack/legacy/plugins/siem/public/pages/case/components/user_action_tree/user_action_title.test.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
+import copy from 'copy-to-clipboard';
+import { Router, routeData, mockHistory } from '../__mock__/router';
+import { caseUserActions as basicUserActions } from '../__mock__/case_data';
+import { UserActionTitle } from './user_action_title';
+import { TestProviders } from '../../../../mock';
+
+const outlineComment = jest.fn();
+const onEdit = jest.fn();
+const onQuote = jest.fn();
+
+jest.mock('copy-to-clipboard');
+const defaultProps = {
+  createdAt: basicUserActions[0].actionAt,
+  disabled: false,
+  fullName: basicUserActions[0].actionBy.fullName,
+  id: basicUserActions[0].actionId,
+  isLoading: false,
+  labelEditAction: 'labelEditAction',
+  labelQuoteAction: 'labelQuoteAction',
+  labelTitle: <>{'cool'}</>,
+  linkId: basicUserActions[0].commentId,
+  onEdit,
+  onQuote,
+  outlineComment,
+  updatedAt: basicUserActions[0].actionAt,
+  username: basicUserActions[0].actionBy.username,
+};
+
+describe('UserActionTitle ', () => {
```

Comment:
Shouldn't we test the different types of titles that can be produced by the component?

Reply:
these tests are done in the helper functions to avoid rendering the full action title every time